### PR TITLE
Milestone 3: Final Touches

### PR DESCRIPTION
**Pull request by fabgrel10**
In this PR:

- [x] Improve the look and feel of the application
- [x] Keep the general layout of the wireframe
- [x] Add responsive design
- [x] Use plain CSS

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @fabgrel10 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/scss/app.scss [hunk 1]
```diff
@@ -83,7 +94,7 @@ li:nth-child(odd) {
   margin: 25px 0;
 }
 
-/* Left section */
+/* Right section */
 /* stylelint-disable-next-line selector-id-pattern */
```
Comment (mikenath223) [inline src/scss/app.scss:98]: - [ ] Kindly remove this stylelint disable line. It's best to fix the issue in the code.

File: src/app.js [hunk 2]
```diff
@@ -1,32 +1,28 @@
...
-    score: score.value,
-  };
-
-  if (newScore.name && newScore.score) {
-    scores.push(newScore);
-    addScoreToLocalStorage(scores);
-    renderScoreboard(scores);
+  if (newScore.user && newScore.score) {
+    postScore(newScore.setScore());
+    // getScores(); // This refreshes the scoreboard automatically when a new score is added.
```
Comment (mikenath223) [inline src/app.js:18]: - [ ] Please remove this commented code.

Comment (fabgrel10) [reply src/app.js]: @mikenath223 all done.

**Review by ArthurGC — APPROVED** [review]
# Approved project!  💯 ✅ 🥳 🎈 🎉
Hi  @fabgrel10 

Amazing job implementing suggestions from the previous reviewer! 💯 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**